src/lcals/FIRST_MIN-Sycl.cpp

```diff
@@ -84,23 +84,24 @@ void FIRST_MIN::runSyclVariantImpl(VariantID vid)
 
   } else if ( vid == RAJA_SYCL ) {
 
-    using VL_TYPE = RAJA::expt::ValLoc<Real_type>;
-
     startTimer();
     for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
 
-       VL_TYPE tloc(m_xmin_init, m_initloc);
+       RAJA::expt::ValLoc<Real_type, RAJA::Index_type> tminloc(m_xmin_init,
+                                                               m_initloc);
 
        RAJA::forall< RAJA::sycl_exec<work_group_size, true /*async*/> >(
          res,
          RAJA::RangeSegment(ibegin, iend), 
-         RAJA::expt::Reduce<RAJA::operators::minimum>(&tloc),
-         [=]  (Index_type i, VL_TYPE& loc) {
-           loc.min(x[i], i);
+         RAJA::expt::Reduce<RAJA::operators::minimum>(&tminloc),
+         [=]  (Index_type i,
+           RAJA::expt::ValLocOp<Real_type, Index_type,
```

Comment:
Should check matching `Index_type` here too.

Reply:
Fixed.